modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -158,278 +160,288 @@ Deploying myMuleApp ...
-* 16 vCores workers have an additional 320 GB of storage
+.The arrow shows the configuration tabs on the *Deploy Application* page.
+image::cloudhub-deploy-app-config-tabs.png[Configuration tabs on the Deploy Application page]
 
-[IMPORTANT]
---
-The workers with additional storage listed above are available only for Mule version 3.6.2 or later, or API gateway version 2.0.2 or later.
-+
-It is recommended to access the additional storage at /tmp.
---
 
-Select the options from the drop-down menu to configure the computing power you need:
+* <<Runtime>>
+* <<Autoscaling>>
+* <<Properties>>
+* <<Insight>>
+* <<Logging>>
+* <<Static IPs>>
 
-image::workersizeandqty.png[WorkerSizeAndQty]
 
-Depending on how many vCores are allocated to your account, some of these options may not be applicable, as you may not have enough available capacity.
 
-[IMPORTANT]
-If you select more vCores than are available in your account, CloudHub allows you to create the application using the console, but you cannot start your application until the vCores are available.
+=== Runtime
 
-When deploying an application with more than one worker, CloudHub automatically balances any incoming traffic load across your allocated workers. For more information, see xref:cloudhub-fabric.adoc[Clustering].
+The *Runtime* tab includes the following settings:
 
-[[region]]
-=== Region
+Runtime version::
+The runtime version must the same Mule version used to develop your application.
++
+Once the app is deployed, CloudHub automatically alerts you when updates for the selected runtime version are available. 
+See xref::cloudhub-app-runtime-version-updates.adoc[Updating the Runtime Version for CloudHub Apps].
 
-If you have global deployment enabled on your account, you can change the *Region* to which your application deploys using the drop-down menu. Administrators can xref:managing-cloudhub-specific-settings.adoc[set the default region] on the *Organization* tab in Account Settings, but that region can be adjusted here when the application is deployed, if necessary.
+Worker size::
+See <<Worker Size and vCores>>.
+Workers::
+When deploying an application with more than one worker, CloudHub automatically balances any incoming traffic load across your allocated workers.
+See xref:cloudhub-fabric.adoc[Clustering].
 
-[NOTE]
+Region::
+If you have global deployment enabled on your account, you can change the region to deploy your application to.
++
 --
-* Applications deployed to Europe automatically have their domain updated to  `+http://myapplicationdomain.eu.cloudhub.io+`.
-* Applications deployed to Asia / Pacific automatically have their domain updated to `+http://myapplicationdomain.au.cloudhub.io+`.
+* For applications deployed to Europe, the domain is automatically updated to `+http://myapplicationdomain.eu.cloudhub.io+`
+* For applications deployed to Asia-Pacific, the domain is automatically updated to `+http://myapplicationdomain.au.cloudhub.io+`
```

Comment:
Those domain updates only applies to a few cases. In most cases, the domain will automatically be updated whenever an application is moved to another region based on the mapping in this region-dns record table https://docs.mulesoft.com/runtime-manager/cloudhub-networking-guide#regional-services